pkg/virt-controller/watch/vm_test.go

```diff
@@ -778,19 +776,25 @@ var _ = Describe("VirtualMachine", func() {
 
 			existingDataVolume.Namespace = "default"
 			existingDataVolume.Status.Phase = cdiv1.Succeeded
+
+			vm, err := virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Create(context.TODO(), vm, metav1.CreateOptions{})
+			Expect(err).To(Succeed())
 			addVirtualMachine(vm)
+
```

Comment:
nit: empty line

Reply:
Intentional to distinguish where the vm is configured and then what next needs to be prepared.